kubernetes/loculus/values.yaml

```diff
@@ -96,6 +96,16 @@ defaultOrganismConfig: &defaultOrganismConfig
         noInput: true
         perSegment: true
         oneHeader: true
+      - name: geoLocLatitude
+        displayName: Latitude
+        header: Sample details
+        type: float
+        guidance: Geo-coordinate latitude in decimal degree (WGS84) format, i.e. values in range -90 to 90.
```

Comment:
I think positive is north, maybe worth adding again so people don't need to look up (I'm not sure we use the guidance yet but I hope we will soon)

Reply:
I really don't think that we need to add this: the standard definition of latitude and longitude is
- longitude: positive numbers are east of the Prime Meridian and negative numbers are west of it
- latitude positive values are north of the equator, negative values are south of the equator. 

So we can add this but if someone is adding data in this field they will be using a standard tool for calculating coordinates anyways - as long as we say we want decimal degree it should be clear.